cypress/e2e/share.spec.js

```diff
@@ -95,16 +95,17 @@ describe('Open test.md in viewer', function() {
 			})
 	})
 
-	it('Shares a folder as a public read only link', function() {
+	it.only('Shares a folder as a public read only link', function() {
 		cy.shareFile('/folder')
 			.then((token) => {
-				cy.logout()
-				cy.visit(`/s/${token}`)
+				return cy.logout()
+					.then(() => token)
 			})
+			.then((token) => cy.visit(`/s/${token}`))
 			.then(() => {
-				cy.get('#rich-workspace').should('contain', 'Hello world')
+				cy.get('[data-text-el="editor-content-wrapper"]').should('contain', 'Hello world')
 				cy.openFile('test.md')
-				cy.get('#editor-container').should('be.visible')
+				cy.get('[data-text-el="editor-content-wrapper"]').should('be.visible')
 				cy.get('#editor .ProseMirror').should('contain', 'Hello world')
 				cy.get('#editor .ProseMirror h2').should('contain', 'Hello world')
```

Comment:
I think we should make sure that we are actually looking at the modal and not the workspace here.